rinad/src/ipcm/main.cc

```diff
@@ -65,10 +67,14 @@ void handler(int signum)
 			rinad::IPCManager->stop();
 			break;
 		case SIGCHLD:
-			while (waitpid(WAIT_ANY, NULL, WNOHANG) > 0) {
-				LOG_DBG("Child IPC Process Daemon died, removed from process table");
+			while (pid = waitpid(WAIT_ANY, NULL, WNOHANG), pid > 0) {
+				LOG_DBG("Child IPC Process Daemon %d died, removed from process table",
+					pid);
```

Comment:
This is correct, but I personally find the comma operator very bug prone. Anyway,  if you have tested it and it worked it is ok to me.